doc/src/manual/noteworthy-differences.md

```diff
@@ -158,6 +158,13 @@ For users coming to Julia from R, these are some noteworthy differences:
   * In Julia, values are not copied when assigned or passed to a function. If a function modifies an array, the changes
     will be visible in the caller. This is very different from R and allows new functions to operate
     on large data structures much more efficiently.
+  * In Julia, a function argument is either positional, or a keyword
+    argument. Unlike in R -- where any argument can be passed either
+    positionally or by name -- in Julia, positional arguments cannot be
```

Comment:
nit: I would use three hyphens for em-dash and remove surrounding spaces, as well as the second redundant "in Julia," so that would read

>  Unlike in R---where any argument can be passed either positionally or by name---positional arguments...

Reply:
Ah yes great, thank you, will edit.